CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+X.Y.Z Release notes
+=============================================================
+### Compatibility
+* Sync protocol: 24
+* Server-side history format: 4
+* File format: 7
+* Realm Object Server: 3.0.0 or later
+
+### Breaking changes
+* None.
+
+### Enhancements
+* None.
+
+### Bug fixes
+* [Sync] Fixed a bug which crash query-based Realms (realm-js-private #450).
```

Comment:
Is it possible to make it more explicit what the bug is since the issue is private?